ui/mirage/config.js

```diff
@@ -443,7 +443,7 @@ export default function () {
     return JSON.stringify(findLeader(schema));
   });
 
-  this.get('/acl/tokens', function ({tokens}, req) {
+  this.get('/acl/tokens', function ({ tokens }, req) {
```

Comment:
`question`: Is this a prettier change? Happens to me a lot and I end up needing to use `git add -p` to patch add the changes.